tsl/src/continuous_aggs/refresh.c

```diff
@@ -622,7 +586,6 @@ emit_up_to_date_notice(const ContinuousAgg *cagg, const CaggRefreshCallContext c
 {
 	switch (callctx)
 	{
-		case CAGG_REFRESH_CHUNK:
```

Comment:
Is it not used?

Reply:
It was only used here. Therefore, I also removed it from the `CaggRefreshCallContext` enum.